src/style.css

```
@@ -0,0 +1,39 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: start;
+}
+
+.stars__star {
+  background: url("images/star.svg") no-repeat center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(5n) {
+  background: url("images/star-active.svg") no-repeat center;
+}
+
+.stars--2 .stars__star:nth-child(1n + 4) {
+  background: url("images/star-active.svg") no-repeat center;
+}
```

```suggestion
.stars--1 .stars__star:nth-child(5n),
.stars--2 .stars__star:nth-child(1n + 4) {
  background: url("images/star-active.svg") no-repeat center;
}
```
you can unite all this selectors like this